tests/data/contracts.py

```diff
@@ -138,3 +138,508 @@
     return (arg,);
 }
 """
+CAIRO_BINDINGS_CONTRACT_ENUM = """\
```

Comment:
I would put cairo1 contracts in a separate file to avoid confusion

Reply:
[move cairo 1 contracts to a separate file](https://github.com/software-mansion/protostar/pull/1316/commits/1c44c3a48da4ed718c609b7cdf1d06019fa3288f)